lib/documents/schemas/utaac_decisions.json

```diff
@@ -17,7 +17,7 @@
   ],
   "subscription_list_title_prefix": "Upper Tribunal Administrative Appeals Chamber decisions",
   "pre_production": false,
-  "summary": "<p>Check the Courts and Tribunals Judiciary website for <a href=\"http://administrativeappeals.decisions.tribunals.gov.uk/Aspx/default.aspx\">decisions made before January 2016</a>.</p>",
+  "summary": "<p>Find decisions on appeals to the Upper Tribunal (Administrative Appeals Chamber), including social security and child support appeals.</p><p>Includes decisions after December 2015. Find details of <a rel=\"external\" href=\"http://www.osscsc.gov.uk/Aspx/default.aspx\">older decisions.</a></p>",
```

Comment:
http://www.osscsc.gov.uk/Aspx/default.aspx\ actually redirects to http://administrativeappeals.decisions.tribunals.gov.uk/Aspx/default.aspx

So we should probably just have the later.